hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1203,11 +1203,41 @@ message UpdateGetS3SecretRequest {
     required string awsSecret = 2;
 }
 
+message OMNodeInfo {
+    required string nodeId = 1;
+    required string hostAddress = 2;
+    required uint32 rpcPort = 3;
```

Comment:
We are not using rpcPort, httpAddr, httpsAddr, serviceId these can be removed from this.

Reply:
Done.